ui/src/layouts/storage/mod.rs

```diff
@@ -299,20 +335,136 @@ pub fn FilesLayout<'a>(cx: Scope<'a, Props<'a>>) -> Element<'a> {
                         },
                     }
                  )
+                },
+                if *show_modal_to_select_chats_to_send_files.get()  {
+                    rsx!(
+                        div {
+                            class: "send-files-to-several-chats-div",
+                            Modal {
+                                open: *show_modal_to_select_chats_to_send_files.clone(),
+                                transparent: false,
+                                onclose: move |_| show_modal_to_select_chats_to_send_files.set(false),
+                                div {
+                                    class: "modal-div-files-layout",
+                                    FilesLayout {
+                                        storage_files_to_chat_mode_is_active: show_modal_to_select_chats_to_send_files.clone(),
+                                        select_chats_to_send_files_mode: show_modal_to_select_chats_to_send_files.clone(),
+                                        on_files_attached: move |(files_location, convs_id): (Vec<Location>, Vec<Uuid>)| {
+                                            let warp_cmd_tx = WARP_CMD_CH.tx.clone();
+                                            let (tx, _) = oneshot::channel::<Result<(), warp::error::Error>>();
+                                            let msg = vec!["".to_owned()];
```

Comment:
No need to include an empty string unless it was there intentionally

```suggestion
                                            let msg = vec![];
```